synapse/http/servlet.py

```diff
@@ -104,7 +112,7 @@ def parse_boolean_from_args(args, name, default=None, required=False):
 
 
 def parse_string(request, name, default=None, required=False,
-                 allowed_values=None, param_type="string"):
+                 allowed_values=None, param_type="string", encoding='ascii'):
```

Comment:
can has docstring for encoding please

Reply:
also can you change the `str`s in the docstrings to be `bytes` or `unicode` or both as appropriate?

Reply:
fixed